src/command_modules/azure-cli-network/azure/cli/command_modules/network/dns_zone.py

```diff
@@ -0,0 +1,26 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+"""
+Commands for DNS Zone Management
+"""
+
+#pylint: disable=too-many-arguments
+def create_dns_zone(client, resource_group_name, zone_name, location='global', tags=None,
+                    if_none_match=False):
+    body = {'location': location}
+    if tags:
+        body['tags'] = tags
+
+    kwargs = {
+        'resource_group_name':resource_group_name,
+        'zone_name': zone_name,
+        'parameters': body
+    }
+
+    if if_none_match:
+        kwargs['if_none_match'] = '*'
+
+    return client.create_or_update(**kwargs)
```

Comment:
It worth discuss if we want to adopt this approach. There is obviously benefit here but in the meantime we lose control of deciding what and how CLI expose the command.